doc/source/contributing_code.rst

```diff
@@ -114,15 +119,11 @@ With pip and virtualenv
 
 Like conda, virtual environments managed with `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ allow you to use different versions of Python and Python packages for your different project. Unlike conda, virtualenv is not a general-purpose package manager. Instead, it leverages what is available on your system, and lets you install Python packages using pip.
 
-To use virtual environments you have to install the virtualenv package first. This can be done with either pip or the package manager of your system:
+To use virtual environments you have to install the virtualenv package first. This can be done with pip:
 
     .. code-block:: bash
 
         pip install virtualenv
```

Comment:
Here and everywhere else, I remember someone telling me that using `python -m pip` was actually safer than relying on pip having been picked up properly by your environment. Thoughts on switching everything over to that?

Reply:
Good point. I think if you did not install it explicitly with `conda` you can pick up the system `pip` or the `base` environment one. I'd be in favour of switching (that's how I use it), unsless there are drawbacks I'm unaware of.

Reply:
Unless @lilyminium has any objections I say let's do it.